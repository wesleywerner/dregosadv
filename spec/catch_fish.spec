whip the water
x fish
take it
eat it
smell it
put it in the water
whip the water
