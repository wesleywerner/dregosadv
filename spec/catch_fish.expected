  THIS GAME IS IN DEBUG MODE
  
  Dr Ego and the Egg of Man-Toomba
  An Interactive Fiction Story
  by Special Agent.
  Release 1 / Serial number 400410 / Inform v6.33 Library v6.12.3pre SX
  
  The Maro River (in the canoe)
. 
  In the canoe I can also see a trap (which is closed and empty), a fishing
  spear and a fishing net.
  
> >  *SNAP* -- I crack the whip into the water and snag a fish. "Mogillo!" my
  guide shouts excitedly. "What are the chances!"
  
  A fish flops around inside the canoe.
  
  [I scored my first point! Type SCORE/FULL to list them.]
  
> >  It looks like a black bass.
  
> >  No, I will leave the fish for my guide.
  
  My guide paddles with the oar.
  
> >  (first taking the fish)
  No, I will leave the fish for my guide.
  
> >  I smell fresh air.
  When I am ready, I can sit back and WAIT until we reach the shore.
  
  "We will have fish for dinner tonight", my guide says hungrily.
  
> >  (first taking the fish out of the canoe)
  Ok, I released it back into the Maro River. "Ai, there goes supper," my
  guide says.
  
> >  I crack the whip at the Maro River -- *SNAP*
  
  My guide looks across the salty water, into the distance.
  
> >